src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/utilities/PointerLocation.java

```diff
@@ -304,15 +304,18 @@ public void printOn(PrintStream tty, boolean printAddress, boolean verbose) {
         if (Assert.ASSERTS_ENABLED) {
           Assert.that(isInBlobUnknownLocation(), "Should have known location in CodeBlob");
```

Comment:
Line 305's Assert message is hard to understand: "Should have known location in CodeBlob", when it would accept knowing that the location is unknown.

It means that none of the isInBlobXXX flags are true, and probably impossible to trigger at the moment.
If you ever wanted to delete it, or make it say "Should have known or unknown location in CodeBlob" then I'd agree. 8-)